hr_employee_second_lastname/models/hr_employee.py

```diff
@@ -75,7 +75,6 @@ def _prepare_vals_on_write_firstname_lastname(self, vals):
     def _update_partner_firstname(self):
         for employee in self:
             partners = employee.mapped("user_id.partner_id")
-            partners |= employee.mapped("address_home_id")
```

Comment:
Shouldn't this be replaced by `address_id` instead?

CC @xmglord

Reply:
No it should be replaced with `work_contact_id` since `address_home_id` was split in multiple fields. I took this from Odoo.

Reply:
Thank you for your feedback, I will fix this quickly.
Regards

Reply:
It's done, but the tests are now failing. Should I do something else?